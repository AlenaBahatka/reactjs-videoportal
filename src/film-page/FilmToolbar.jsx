import React from 'react';

function FilmToolbar(props) {
    return (
        <div className="row">
            <div className="col-sm-9">Films by {props.genre} genre</div>
        </div>
    );
}

export default FilmToolbar;