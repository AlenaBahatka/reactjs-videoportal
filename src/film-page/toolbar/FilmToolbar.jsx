import React from 'react';

function FilmToolbar(props) {
    return (
        <div className="row">
            <div className="col-sm-9">Films by <b>{props.genre}</b> genre</div>
        </div>
    );
}

export default FilmToolbar;