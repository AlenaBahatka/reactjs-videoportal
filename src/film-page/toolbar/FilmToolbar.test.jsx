import React from 'react';
import { shallow } from 'enzyme';
import FilmToolbar from './FilmToolbar';

describe('FilmToolbar', () => {
	it('should correctly render component', () => {
		let filmToolbarComponent = shallow(<FilmToolbar genre="drama" />);

		expect(filmToolbarComponent).toMatchSnapshot();
	});
});
