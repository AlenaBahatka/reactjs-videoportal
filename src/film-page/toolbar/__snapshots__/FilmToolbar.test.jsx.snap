// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmToolbar should correctly render component 1`] = `
<div
  className="row"
>
  <div
    className="col-sm-9"
  >
    Films by 
    <b>
      drama
    </b>
     genre
  </div>
</div>
`;
